superset/models/superset_core/user.py

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import logging
+
+from flask_appbuilder import Model
+from flask_appbuilder.security.sqla.models import assoc_user_role, User as FabUser
+from sqlalchemy.orm import relationship
+
+metadata = Model.metadata  # pylint: disable=no-member
+logger = logging.getLogger(__name__)
+
+
+class SupersetUser(FabUser):
+    __tablename__ = "ab_user"
+
+    __table_args__ = {"extend_existing": True}
+    roles = relationship(
```

Comment:
why defined again? why the SupersetUser model should be aware that you decided to extend another model?